[2021-12-16 13:01:06,842] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse_kym 2021-12-16T13:01:04.961857+00:00 [queued]>
[2021-12-16 13:01:06,863] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse_kym 2021-12-16T13:01:04.961857+00:00 [queued]>
[2021-12-16 13:01:06,864] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-16 13:01:06,864] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-12-16 13:01:06,864] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-16 13:01:06,876] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): cleanse_kym> on 2021-12-16T13:01:04.961857+00:00
[2021-12-16 13:01:06,880] {standard_task_runner.py:52} INFO - Started process 13797 to run task
[2021-12-16 13:01:06,882] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'cleansing_dag', 'cleanse_kym', '2021-12-16T13:01:04.961857+00:00', '--job-id', '33', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/kym_cleansing.py', '--cfg-path', '/tmp/tmp609x6trm', '--error-file', '/tmp/tmpgvwcf9zu']
[2021-12-16 13:01:06,883] {standard_task_runner.py:77} INFO - Job 33: Subtask cleanse_kym
[2021-12-16 13:01:06,922] {logging_mixin.py:104} INFO - Running <TaskInstance: cleansing_dag.cleanse_kym 2021-12-16T13:01:04.961857+00:00 [running]> on host f46d95c5a5d9
[2021-12-16 13:01:06,986] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=cleansing_dag
AIRFLOW_CTX_TASK_ID=cleanse_kym
AIRFLOW_CTX_EXECUTION_DATE=2021-12-16T13:01:04.961857+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-16T13:01:04.961857+00:00
[2021-12-16 13:04:14,642] {python.py:118} INFO - Done. Returned value was: None
[2021-12-16 13:04:14,656] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=cleansing_dag, task_id=cleanse_kym, execution_date=20211216T130104, start_date=20211216T130106, end_date=20211216T130414
[2021-12-16 13:04:14,726] {local_task_job.py:146} INFO - Task exited with return code 0
